checkoutmanager/runner.py

```diff
@@ -82,6 +82,25 @@ def execute_action(dirinfo, custom_actions, action):
         return e
 
 
+def run_one(action, directory=None, url=None, conf=None):
+    custom_actions = get_custom_actions()
+    if not conf:
+        conf = config.Config(os.path.expanduser(CONFIGFILE_NAME))
+    from .dirinfo import DirInfo
```

Comment:
I thought there may be a recursive import, and I suppose also too lazy to check. I'll take a closer look in a little while and move the import to the module level if there's nothing recursive involved.